content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -65,18 +73,26 @@ The Helm chart ships with reasonable default configuration options which are exp
 | `global.arch.ppc64le` | Specifies the scheduling policy for ppc64le architectures| `2` |
 | `galley.enabled` | Specifies whether Galley should be installed for server-side config validation. Requires k8s >= 1.9 | `false` |
 
-The Helm chart also offers significant customization options per individual service. Customize these per-service options at your own risk. The per-service options are exposed via the Parameters tab.
+Passive voice issue:
```

Comment:
Do not add the fact that there was a passive voice issue to the document. Be careful when copy-pasting :smiley:

Reply:
:-)

Reply:
see the new commit 4ab4781